pkg/fission-cli/spec.go

```diff
@@ -343,6 +343,27 @@ func (fr *FissionResources) validate() error {
 			packages[mapKey(pkgMeta)] = true
 		}
 
+		client := util.GetApiClient(c.GlobalString("server"))
+		for _, cm := range f.Spec.ConfigMaps {
+			c, err := client.ConfigMapGet(&metav1.ObjectMeta{
+				Name:      cm.Name,
+				Namespace: cm.Namespace,
+			})
+			if c == nil && err != nil {
```

Comment:
The error may not equal to non-exists error, we need to check it before printing log.
```
if k8serrors.IsNotFound(err) {
   .....
```